build.gradle

```diff
@@ -33,24 +31,32 @@ publishing {
     }
     publications {
         mavenJar(MavenPublication) {
-            artifact bootWar
+            artifact bootJar
```

Comment:
This change removes the bootWar instead of adding the bootJar. Update this to publish both the bootWar and bootJar.